### ISSUE TITLE
Enable floating point operations

### ISSUE THREAD

**Issue by amit-singhs** [amit-singhs/test-elm#9]

**Comment by renovatorruler** [comment]
Status of this ticket?

**Comment by amit-singhs** [comment]
Yeah it is in progress. I have Managed to construct both the numbers in float. Now I'm creating a function to take float as input yield the result in as 'NumberType'. (The type we defined.)
